packages/fs-detectors/src/detectors/filesystem.ts

```diff
@@ -96,12 +96,13 @@ export abstract class DetectorFilesystem {
   /**
    * Writes a file to the filesystem cache.
    * @param name the name of the file to write
-   * @param content The content of the file
+   * @param exists true if the files exists, false otherwise
```

Comment:
The comment here needs updating because I didn't understand the purpose until reading the PR description. Also it doesn't mention what happens in the undefined case.

Reply:
Refactored to make `content` a required parameter again, this comment should still hold 🤞